tests/pending/pos/i2797/Fork_1.java

```diff
@@ -0,0 +1,7 @@
+// Test is pending because we have no good way to test it.
```

Comment:
I don't understand why this test is pending, since Fork is suffixed with 1 and Test with 2, they should be compiled as you want